vector-im/element-android
# [Location Share] - Wrong room live location status bar visibility in timeline (PSG-625)

## Type of change


## Content


Fixing the visibility of the top status bar in a room timeline which indicates whether there is an active live location shared from user in this room.

## Motivation and context


Closes #6537 

## Screenshots / GIFs




TODO

## Tests



- Start a live location share in a room
- Go back to room list
- Navigate again in the room where there is an active live location share
- Check the top status bar is visible
- Go to another room
- Check the top status bar is not displayed
- Start a new live location in this room
- Check the top status bar is visible in this room
- Check live can be stopped from status bar in both rooms where there is an active live

## Tested devices

- OS version(s): Android 11

## Checklist
